docs/development/README.adoc

```diff
@@ -28,6 +28,41 @@ go get -u github.com/gogo/protobuf/protoc-gen-gogoslick
 Run `+go generate ./.../+` from `keep-core` and if everything is fine, start the
 Keep client with `go run main.go`.
 
```

Comment:
What do you think about moving `Setting up local Keep network` part between `Building` and `Quick Installation`? 

The developer needs to have their accounts set up and Ethereum client running to execute `Quick Installation` and this part is covered by `Setting up local Keep network`.

Reply:
Sure, makes sense